met-web/src/components/engagement/view/ActionContext.tsx

```diff
@@ -233,12 +206,11 @@ export const ActionProvider = ({ children }: { children: JSX.Element | JSX.Eleme
     }, [slug]);
 
     useEffect(() => {
-        fetchEngagement();
+        // revalidator.revalidate();
```

Comment:
Some more commented out code in this file - necessary going forward?

Reply:
Nope, it can be removed.